### ISSUE TITLE
Optimize only one strategy  : invalid pointer access in 'EA31337.mq4' (5424,7)

### ISSUE THREAD

**Issue by soumsena** [EA31337/EA31337#117]
@kenorb , when I activate a single strategy for example **BAND** to test and optimize, I have an error
**invalid pointer access in 'EA31337.mq4' (5424,7)** thanks

Line:
```
if (trade[M1].Chart().GetBarTime() == last_action_time) return;
```

**Comment by kenorb** [comment]
Fixed in 7894c26. It's going to be merged to master soon when I do more testing.

**Comment by kenorb** [comment]
@soumsena Please test the new released version at https://github.com/EA31337/EA31337/releases/tag/v1.076
